.vitepress/components/TagChip.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+
+import {TAG_PROP} from "../theme/enhancements/i18n/utility/tags.ts";
```

Comment:
missing?

Reply:
I think this happened because of the revert-commits. Will fix it!

Reply:
It's back!